documentation/book/con-metrics-kafka-mini-kube-shift.adoc

```diff
@@ -0,0 +1,17 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-setup.adoc
+[id='con-metrics-kafka-mini-kube-shift-{context}']
+
+= Using metrics with `minikube` or `minishift`
```

Comment:
@ppatierno Do you think this is still valid? Can anyone run Strimzi with 2GB Minikube? shouldn't we just remove this? WDYT?

Reply:
Tbh I always use `minishift` which has 4GB as default while `minikube` has 2 GB as default. For having prometheus and grafana I think it's still valid having more memory.

Reply:
You actually use 4GB minishift? Wow. I'm surprised that works ... ok, let's keep it here then.